custom/icds/messaging/indicators.py

```diff
@@ -227,3 +237,71 @@ def convert_to_date(date):
             messages.append(self.render_template(context, language_code=language_code))
 
         return messages
+
+
+class LSAggregatePerformanceIndicator(LSIndicator):
+    template = 'ls_aggregate_performance.txt'
+
+    @property
+    @memoized
+    def restore_user(self):
+        return OTARestoreCommCareUser(self.domain, self.user)
+
+    @property
+    @memoized
+    def report_configs(self):
+        report_ids = [
+            HOME_VISIT_REPORT_ID,
+            THR_REPORT_ID,
+            CHILDREN_WEIGHED_REPORT_ID,
+            DAYS_AWC_OPEN_REPORT_ID,
+        ]
+        app = get_app(self.domain, SUPERVISOR_APP_ID)
+        return {
+            report_config.report_id: report_config
+            for module in app.modules if isinstance(module, ReportModule)
+            for report_config in module.report_configs
+            if report_config.report_id in report_ids
+        }
+
+    def get_report_fixture(self, report_id):
+        return ReportFixturesProvider._report_config_to_fixture(
```

Comment:
nit: might make sense to remove the `_` from the signature now that it's being called from other places